packages/models-library/src/models_library/api_schemas_webserver/storage.py

```diff
@@ -1,6 +1,7 @@
 from pathlib import Path
 from typing import Annotated, Any
 
+from aiohttp import web
```

Comment:
model_library must not have this dependency! Move the adapter below `from_job_id` to the webwerver